calendar_reboot.js

```diff
@@ -226,31 +223,31 @@ $(document).ready(function() {
         //the name from the saved calendars dropdown
         
         //when you delete a calendar, it needs to be removed from local Storage
-        //it's unique Id needs to be removed from the calendarUniqueId dictionary
+        //it's unique Id needs to be removed from the allCalendarIds dictionary
         //it's name needs to be removed from the saved calendars dropdown
         //clear the calendar from the page and uncollapse build calendar
         
         //need to check if there is a calendar loaded that needs to be deleted.
```

Comment:
Comments that are instructions for future you should begin with `TODO:` or `TODO(@andrayantelo):` if it should be known who is responsible for doing it.